compiler/luci/import/src/CircleReader.cpp

```diff
@@ -266,7 +266,10 @@ void copy_tensor_attributes(const circle::TensorT &tensor, CircleNode *node)
   node->rank(dims.size());
   for (uint32_t r = 0; r < dims.size(); ++r)
   {
-    node->dim(r) = loco::Dimension(dims[r]);
+    if (tensor.shape_signature.size() > 0 && tensor.shape_signature.at(r) == -1)
```

Comment:
```suggestion
    if (tensor.shape_signature.size() == dims.size() && tensor.shape_signature.at(r) == -1)
```
more safe?

Reply:
Hmm.. if then, it is more safe to add assert because size of shape_signature should be always 0 or `dims.size()`.